src/Chartjs/assets/dist/controller.js

```diff
@@ -20,6 +20,10 @@ class default_1 extends Controller {
         if (!(this.element instanceof HTMLCanvasElement)) {
             throw new Error('Invalid element');
         }
+        if(Chart.getChart(this.element)) {
```

Comment:
Maybe to check before lines 14/19 ? Or the thrown event could be missleading .. wdyt ?

Reply:
Sure! I'll update it 👍🏻